types/react-google-recaptcha/react-google-recaptcha-tests.tsx

```diff
@@ -20,7 +20,15 @@ const invisibleRecaptcha: React.FC = () => {
     );
 };
 
-const basicRecapchta2 = <ReCAPTCHA2 ref={handleRef} sitekey="xxx" onChange={a => a} className="mockclass" />;
+const basicRecapchta2 = (
```

Comment:
suggestion: Instead of modifying the basic test, would you create an additional test for this optional prop like `isolatedReacaptcha2`

Reply:
I agree with this ☝️

Reply:
Sure np, made the change in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68245/commits/a680640d063de41ba2e71001a71cdf42c157e92d